tests/test_git.py

```diff
@@ -114,11 +113,4 @@ def test_overview_timeseries_with_df(self):
         authors_test = pd.read_csv(AUTHORS_BY_PERIOD)
         self.assertIsInstance(authors, pd.DataFrame)
         assert_array_equal(authors_test['value'], authors['value'])
-
-    @classmethod
-    def tearDownClass(cls):
-        """
-        Destroy the infrastructure created for unittests
-        """
-
-        teardown_data_source(cls.git[0], cls.git[1])
+        assert_array_equal(authors_test['value'], authors['value'])
```

Comment:
Why do you need to check this twice? The previous line checks the same.

Reply:
Thanks for pointing this out!